content/v2.0/reference/flux/stdlib/experimental/http/_index.md

```diff
@@ -0,0 +1,31 @@
+---
+title: Flux HTTP package
+list_title: HTTP package
+description: >
+  The Flux Experimental HTTP package provides experimental functions for transferring
+  data using the HTTP protocol.
+  Import the `experimental/http` package.
+menu:
+  v2_0_ref:
+    name: HTTP
+    identifier: HTTP-exp
+    parent: Experimental
+weight: 201
+v2.0/tags: [functions, http, package]
+---
+
+The Flux Experimental HTTP package provides experimental functions for transferring
+data using the HTTP protocol.
+
+{{% warn %}}
+The experimental HTTP package is subject to change at any time.
+By using this package, you accept the [risks of experimental functions](/v2.0/reference/flux/stdlib/experimental/#use-experimental-functions-at-your-own-risk).
+{{% /warn %}}
```

Comment:
We might want to make a standard shortcode warning for experimental packages:

```
{{% warn %}}
This package is experimental and subject to change at any time.
By using this package, you accept the [risks of experimental functions](/v2.0/reference/flux/stdlib/experimental/#use-experimental-functions-at-your-own-risk).
{{% /warn %}}
```

Is there a way to include page titles as variables in those?

Reply:
This warning is only used a handful of times and is unique to each pages it's on, so I don't feel it needs a shortcode.